File: cc/007.cc
// Project Euler Problem 007
//
// Find the 10001st prime.

#include <cstdio>
#include "primes1.h"

int main(int argc, const char* argv[]) {
  Primes1<long> primes;
  printf("%ld\n", primes.get(10000));
}

File: go/problems/007.go
// Project Euler Problem 007
//
// Find the 10001st prime.

package problems

import (
	"fmt"

	"github.com/zellyn/project-euler/go/primes"
)

func Problem007() string {
	return fmt.Sprint(primes.Get(10000))
}

func init() {
	Functions[7] = Problem007
}
